data/operators.csv

```diff
@@ -420,3 +420,4 @@ Bryan Barbolina trading as Cloudwebservices,cloud,signed + filtering,safe,213268
 Hazelnet,ISP,signed + filtering,safe,200242,75603
 Parknet,ISP,signed + filtering,safe,197301,19561
 Kerfuffle,Cloud,signed + filtering,safe,35008,11481
+Rozint Ltd Co,ISP,signed + filtering,safe,21738,534
```

Comment:
The AS rank needs an update: https://asrank.caida.org/asns?asn=21738&type=search
```suggestion
Rozint Ltd Co,ISP,signed + filtering,safe,21738,2442
```